plugins/@grouparoo/clickhouse/src/lib/query-import/connection.ts

```diff
@@ -0,0 +1,14 @@
+import { buildConnection } from "@grouparoo/app-templates/dist/source/query";
+import { executeQuery } from "@grouparoo/mysql/dist/lib/query-import/executeQuery";
+import { getChangedRows } from "@grouparoo/mysql/dist/lib/query-import/getChangedRows";
+
+export function getConnection() {
+  return buildConnection({
+    app: "clickhouse",
```

Comment:
Once you rebase from `main`, this will become 

```suggestion
    apps: ["clickhouse"],
```